tests/run_model_templates_tests.sh

```diff
@@ -1,4 +1,4 @@
-virtualenv venv || exit 1
+virtualenv venv -p python3 || exit 1
```

Comment:
Why don't we finally get rid of this `virtualenv` and use `venv`?

Reply:
I didn't see much difference, though it exists for sure. I started to use virtualenv and have been still using it. I dont mind. Replaced.

Reply:
Reverted. Not this time. I will do it as soon as new docker image on ci is available.